INFO - 2024-05-03 08:37:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-05-03 08:57:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-05-03 08:57:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-05-03 09:06:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-05-03 09:06:48 --> mysqli_sql_exception: Unknown column 'codigo_usuario' in 'where clause' in C:\xampp\htdocs\ProyectoDaw\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\ProyectoDaw\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT * FROM i...', 0)
#1 C:\xampp\htdocs\ProyectoDaw\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT * FROM i...')
#2 C:\xampp\htdocs\ProyectoDaw\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT * FROM i...')
#3 C:\xampp\htdocs\ProyectoDaw\app\Models\DataBaseHandler.php(130): CodeIgniter\Database\BaseConnection->query('SELECT * FROM i...', Array)
#4 C:\xampp\htdocs\ProyectoDaw\app\Models\DataBaseHandler.php(106): App\models\DataBaseHandler->jls_check_participant_exists(NULL, NULL)
#5 C:\xampp\htdocs\ProyectoDaw\app\Controllers\Home.php(90): App\models\DataBaseHandler->jls_add_new_participant('Juliano', NULL, NULL)
#6 C:\xampp\htdocs\ProyectoDaw\system\CodeIgniter.php(943): App\Controllers\Home->add_new_participant()
#7 C:\xampp\htdocs\ProyectoDaw\system\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\ProyectoDaw\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\ProyectoDaw\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-05-03 09:06:48 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'codigo_usuario' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 APPPATH\Models\DataBaseHandler.php(130): CodeIgniter\Database\BaseConnection->query('SELECT * FROM inscripciones WHERE id_torneo = ? AND codigo_usuario = ?', [...])
 2 APPPATH\Models\DataBaseHandler.php(106): App\models\DataBaseHandler->jls_check_participant_exists(null, null)
 3 APPPATH\Controllers\Home.php(90): App\models\DataBaseHandler->jls_add_new_participant('Juliano', null, null)
 4 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Home->add_new_participant()
 5 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-05-03 09:06:48 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'codigo_usuario' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT * FROM inscripciones WHERE id_torneo = NULL AND codigo_usuario = NULL')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT * FROM inscripciones WHERE id_torneo = NULL AND codigo_usuario = NULL')
 3 APPPATH\Models\DataBaseHandler.php(130): CodeIgniter\Database\BaseConnection->query('SELECT * FROM inscripciones WHERE id_torneo = ? AND codigo_usuario = ?', [...])
 4 APPPATH\Models\DataBaseHandler.php(106): App\models\DataBaseHandler->jls_check_participant_exists(null, null)
 5 APPPATH\Controllers\Home.php(90): App\models\DataBaseHandler->jls_add_new_participant('Juliano', null, null)
 6 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Home->add_new_participant()
 7 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-05-03 09:06:48 --> [Caused by] mysqli_sql_exception: Unknown column 'codigo_usuario' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT * FROM inscripciones WHERE id_torneo = NULL AND codigo_usuario = NULL', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT * FROM inscripciones WHERE id_torneo = NULL AND codigo_usuario = NULL')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT * FROM inscripciones WHERE id_torneo = NULL AND codigo_usuario = NULL')
 4 APPPATH\Models\DataBaseHandler.php(130): CodeIgniter\Database\BaseConnection->query('SELECT * FROM inscripciones WHERE id_torneo = ? AND codigo_usuario = ?', [...])
 5 APPPATH\Models\DataBaseHandler.php(106): App\models\DataBaseHandler->jls_check_participant_exists(null, null)
 6 APPPATH\Controllers\Home.php(90): App\models\DataBaseHandler->jls_add_new_participant('Juliano', null, null)
 7 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Home->add_new_participant()
 8 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-05-03 09:09:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-05-03 09:09:19 --> mysqli_sql_exception: Column 'id_torneo' cannot be null in C:\xampp\htdocs\ProyectoDaw\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\ProyectoDaw\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `in...', 0)
#1 C:\xampp\htdocs\ProyectoDaw\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `in...')
#2 C:\xampp\htdocs\ProyectoDaw\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `in...')
#3 C:\xampp\htdocs\ProyectoDaw\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `in...', Array, false)
#4 C:\xampp\htdocs\ProyectoDaw\app\Models\DataBaseHandler.php(121): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\ProyectoDaw\app\Controllers\Home.php(90): App\models\DataBaseHandler->jls_add_new_participant('jillo', NULL, NULL)
#6 C:\xampp\htdocs\ProyectoDaw\system\CodeIgniter.php(943): App\Controllers\Home->add_new_participant()
#7 C:\xampp\htdocs\ProyectoDaw\system\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\ProyectoDaw\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\ProyectoDaw\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-05-03 09:09:19 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'id_torneo' cannot be null
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `inscripciones` (`alias`, `id_torneo`, `id_usuario`) VALUES (:alias:, :id_torneo:, :id_usuario:)', [...], false)
 2 APPPATH\Models\DataBaseHandler.php(121): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Home.php(90): App\models\DataBaseHandler->jls_add_new_participant('jillo', null, null)
 4 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Home->add_new_participant()
 5 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-05-03 09:09:19 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'id_torneo' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `inscripciones` (`alias`, `id_torneo`, `id_usuario`) VALUES (\'jillo\', NULL, NULL)')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `inscripciones` (`alias`, `id_torneo`, `id_usuario`) VALUES (\'jillo\', NULL, NULL)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `inscripciones` (`alias`, `id_torneo`, `id_usuario`) VALUES (:alias:, :id_torneo:, :id_usuario:)', [...], false)
 4 APPPATH\Models\DataBaseHandler.php(121): CodeIgniter\Database\BaseBuilder->insert([...])
 5 APPPATH\Controllers\Home.php(90): App\models\DataBaseHandler->jls_add_new_participant('jillo', null, null)
 6 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Home->add_new_participant()
 7 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-05-03 09:09:19 --> [Caused by] mysqli_sql_exception: Column 'id_torneo' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `inscripciones` (`alias`, `id_torneo`, `id_usuario`) VALUES (\'jillo\', NULL, NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `inscripciones` (`alias`, `id_torneo`, `id_usuario`) VALUES (\'jillo\', NULL, NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `inscripciones` (`alias`, `id_torneo`, `id_usuario`) VALUES (\'jillo\', NULL, NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `inscripciones` (`alias`, `id_torneo`, `id_usuario`) VALUES (:alias:, :id_torneo:, :id_usuario:)', [...], false)
 5 APPPATH\Models\DataBaseHandler.php(121): CodeIgniter\Database\BaseBuilder->insert([...])
 6 APPPATH\Controllers\Home.php(90): App\models\DataBaseHandler->jls_add_new_participant('jillo', null, null)
 7 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Home->add_new_participant()
 8 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-05-03 09:09:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-05-03 09:09:32 --> mysqli_sql_exception: Column 'id_torneo' cannot be null in C:\xampp\htdocs\ProyectoDaw\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\ProyectoDaw\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `in...', 0)
#1 C:\xampp\htdocs\ProyectoDaw\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `in...')
#2 C:\xampp\htdocs\ProyectoDaw\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `in...')
#3 C:\xampp\htdocs\ProyectoDaw\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `in...', Array, false)
#4 C:\xampp\htdocs\ProyectoDaw\app\Models\DataBaseHandler.php(121): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\ProyectoDaw\app\Controllers\Home.php(90): App\models\DataBaseHandler->jls_add_new_participant('juliano', NULL, NULL)
#6 C:\xampp\htdocs\ProyectoDaw\system\CodeIgniter.php(943): App\Controllers\Home->add_new_participant()
#7 C:\xampp\htdocs\ProyectoDaw\system\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\ProyectoDaw\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\ProyectoDaw\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-05-03 09:09:32 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'id_torneo' cannot be null
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `inscripciones` (`alias`, `id_torneo`, `id_usuario`) VALUES (:alias:, :id_torneo:, :id_usuario:)', [...], false)
 2 APPPATH\Models\DataBaseHandler.php(121): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Home.php(90): App\models\DataBaseHandler->jls_add_new_participant('juliano', null, null)
 4 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Home->add_new_participant()
 5 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-05-03 09:09:32 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'id_torneo' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `inscripciones` (`alias`, `id_torneo`, `id_usuario`) VALUES (\'juliano\', NULL, NULL)')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `inscripciones` (`alias`, `id_torneo`, `id_usuario`) VALUES (\'juliano\', NULL, NULL)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `inscripciones` (`alias`, `id_torneo`, `id_usuario`) VALUES (:alias:, :id_torneo:, :id_usuario:)', [...], false)
 4 APPPATH\Models\DataBaseHandler.php(121): CodeIgniter\Database\BaseBuilder->insert([...])
 5 APPPATH\Controllers\Home.php(90): App\models\DataBaseHandler->jls_add_new_participant('juliano', null, null)
 6 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Home->add_new_participant()
 7 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-05-03 09:09:32 --> [Caused by] mysqli_sql_exception: Column 'id_torneo' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `inscripciones` (`alias`, `id_torneo`, `id_usuario`) VALUES (\'juliano\', NULL, NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `inscripciones` (`alias`, `id_torneo`, `id_usuario`) VALUES (\'juliano\', NULL, NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `inscripciones` (`alias`, `id_torneo`, `id_usuario`) VALUES (\'juliano\', NULL, NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `inscripciones` (`alias`, `id_torneo`, `id_usuario`) VALUES (:alias:, :id_torneo:, :id_usuario:)', [...], false)
 5 APPPATH\Models\DataBaseHandler.php(121): CodeIgniter\Database\BaseBuilder->insert([...])
 6 APPPATH\Controllers\Home.php(90): App\models\DataBaseHandler->jls_add_new_participant('juliano', null, null)
 7 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Home->add_new_participant()
 8 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-05-03 09:13:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-05-03 09:13:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-05-03 09:13:35 --> ErrorException: Undefined array key "tournament_id"
in APPPATH\Views\registry_participants.php on line 17.
 1 APPPATH\Views\registry_participants.php(17): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "tournament_id"', 'C:\\xampp\\htdocs\\ProyectoDaw\\app\\Views\\registry_participants.php', 17)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\ProyectoDaw\\app\\Views\\registry_participants.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('registry_participants', [], true)
 5 APPPATH\Controllers\Home.php(82): view('registry_participants')
 6 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Home->get_add_participant_page()
 7 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-05-03 09:15:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-05-03 09:16:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-05-03 09:16:10 --> ErrorException: Undefined array key "jls-tournament_id"
in APPPATH\Views\tournament_info.php on line 13.
 1 APPPATH\Views\tournament_info.php(13): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "jls-tournament_id"', 'C:\\xampp\\htdocs\\ProyectoDaw\\app\\Views\\tournament_info.php', 13)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\ProyectoDaw\\app\\Views\\tournament_info.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('tournament_info', [], true)
 5 APPPATH\Controllers\Home.php(103): view('tournament_info', [...])
 6 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Home->get_tournamente_info_page()
 7 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-05-03 09:16:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-05-03 09:17:09 --> ErrorException: Undefined array key "tournament_id"
in APPPATH\Views\registry_participants.php on line 17.
 1 APPPATH\Views\registry_participants.php(17): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "tournament_id"', 'C:\\xampp\\htdocs\\ProyectoDaw\\app\\Views\\registry_participants.php', 17)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\ProyectoDaw\\app\\Views\\registry_participants.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('registry_participants', [], true)
 5 APPPATH\Controllers\Home.php(82): view('registry_participants')
 6 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Home->get_add_participant_page()
 7 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-05-03 09:17:27 --> ErrorException: Undefined array key "jls-tournament_id"
in APPPATH\Views\registry_participants.php on line 17.
 1 APPPATH\Views\registry_participants.php(17): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "jls-tournament_id"', 'C:\\xampp\\htdocs\\ProyectoDaw\\app\\Views\\registry_participants.php', 17)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\ProyectoDaw\\app\\Views\\registry_participants.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('registry_participants', [], true)
 5 APPPATH\Controllers\Home.php(82): view('registry_participants')
 6 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Home->get_add_participant_page()
 7 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-05-03 09:17:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-05-03 09:17:43 --> mysqli_sql_exception: Column 'id_torneo' cannot be null in C:\xampp\htdocs\ProyectoDaw\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\ProyectoDaw\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `in...', 0)
#1 C:\xampp\htdocs\ProyectoDaw\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `in...')
#2 C:\xampp\htdocs\ProyectoDaw\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `in...')
#3 C:\xampp\htdocs\ProyectoDaw\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `in...', Array, false)
#4 C:\xampp\htdocs\ProyectoDaw\app\Models\DataBaseHandler.php(121): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\ProyectoDaw\app\Controllers\Home.php(90): App\models\DataBaseHandler->jls_add_new_participant('sdjfjksdhf', NULL, NULL)
#6 C:\xampp\htdocs\ProyectoDaw\system\CodeIgniter.php(943): App\Controllers\Home->add_new_participant()
#7 C:\xampp\htdocs\ProyectoDaw\system\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\ProyectoDaw\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\ProyectoDaw\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-05-03 09:17:43 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'id_torneo' cannot be null
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `inscripciones` (`alias`, `id_torneo`, `id_usuario`) VALUES (:alias:, :id_torneo:, :id_usuario:)', [...], false)
 2 APPPATH\Models\DataBaseHandler.php(121): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Home.php(90): App\models\DataBaseHandler->jls_add_new_participant('sdjfjksdhf', null, null)
 4 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Home->add_new_participant()
 5 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-05-03 09:17:43 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'id_torneo' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `inscripciones` (`alias`, `id_torneo`, `id_usuario`) VALUES (\'sdjfjksdhf\', NULL, NULL)')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `inscripciones` (`alias`, `id_torneo`, `id_usuario`) VALUES (\'sdjfjksdhf\', NULL, NULL)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `inscripciones` (`alias`, `id_torneo`, `id_usuario`) VALUES (:alias:, :id_torneo:, :id_usuario:)', [...], false)
 4 APPPATH\Models\DataBaseHandler.php(121): CodeIgniter\Database\BaseBuilder->insert([...])
 5 APPPATH\Controllers\Home.php(90): App\models\DataBaseHandler->jls_add_new_participant('sdjfjksdhf', null, null)
 6 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Home->add_new_participant()
 7 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-05-03 09:17:43 --> [Caused by] mysqli_sql_exception: Column 'id_torneo' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `inscripciones` (`alias`, `id_torneo`, `id_usuario`) VALUES (\'sdjfjksdhf\', NULL, NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `inscripciones` (`alias`, `id_torneo`, `id_usuario`) VALUES (\'sdjfjksdhf\', NULL, NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `inscripciones` (`alias`, `id_torneo`, `id_usuario`) VALUES (\'sdjfjksdhf\', NULL, NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `inscripciones` (`alias`, `id_torneo`, `id_usuario`) VALUES (:alias:, :id_torneo:, :id_usuario:)', [...], false)
 5 APPPATH\Models\DataBaseHandler.php(121): CodeIgniter\Database\BaseBuilder->insert([...])
 6 APPPATH\Controllers\Home.php(90): App\models\DataBaseHandler->jls_add_new_participant('sdjfjksdhf', null, null)
 7 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Home->add_new_participant()
 8 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
